app/views/assignments/new.haml

```diff
@@ -1,19 +1,22 @@
 %h1.title New assignment
 = form_for Assignment.new, url: roster_assignments_path(@roster) do |f|
-  %div.container-fluid
-    %div.row
-      %div.col-xs-3
-      %div.col-xs-2
-        %div.form-group
+  .container-fluid
+    .row
+      .col-xs-3
+      .col-xs-2
+        .form-group
           = f.label :user_id
-          = f.select :user_id, @users.pluck(:last_name, :id), class: 'form-control'
-      %div.col-xs-4
-        %div.form-group
+          = f.select :user_id, @users.pluck(:last_name, :id),
```

Comment:
If memory serves, they were introduced in that order. It used to be you had to manually build collections to make your `options_for_select`, but then someone was like "why are we doing this" and so they wrote options *from collection* for select.